README.adoc

```diff
@@ -624,6 +624,10 @@ Having trouble with Eclipse JNoSQL? We’d love to help!
 
 Please report any bugs, concerns or questions with Eclipse JNoSQL to https://github.com/eclipse/jnosql[https://github.com/eclipse/jnosql].
 
+If your issue refers to the https://github.com/eclipse/jnosql-databases[JNoSQL databases project] or
```

Comment:
Great, please, don't forget to add it to the other repositories:
Databases and extentions.